src/net/CUDPComm.cpp

```diff
@@ -1386,15 +1417,18 @@ void CUDPComm::CreateServer() {
     */
 }
 
-OSErr CUDPComm::ContactServer(ip_addr serverHost, port_num serverPort) {
+OSErr CUDPComm::ContactServer(IPaddress &serverAddr) {
     if (noErr == CreateStream(localPort)) {
         long startTime;
         SDL_Event theEvent;
 
+        // Before we "connect", notify the punch server so it can tell the host to open a hole (to our localPort)
+        RequestPunch(serverAddr, localPort);
```

Comment:
question: if PunchServer is not reachable, will this behave as it did before?  I think the answer is yes but wanted to verify.